eng/common/scripts/get-codeowners.ps1

```diff
@@ -1,49 +1,51 @@
 param (
-  $TargetDirectory, # should be in relative form from root of repo. EG: sdk/servicebus
-  $RootDirectory, # ideally $(Build.SourcesDirectory)
-  $VsoVariable = "" # target devops output variable
+  [string]$CodeOwnerPathExpression, # Code path to code owners. e.g sdk/core/azure-amqp
+  [string]$ToolVersion = "", # Placeholder. Will update in next PR
+  [string]$ToolPath = "$env:AGENT_TOOLSDIRECTORY", # The place to check the tool existence. Put $(Agent.ToolsDirectory) as default
+  [string]$DevOpsFeed = "https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-net/nuget/v3/index.json", # DevOp tool feeds.
+  [string]$CodeOwnerFileLocation = "$env:SYSTEM_DEFAULTWORKINGDIRECTORY/.github/CODEOWNERS", # The place we fetch CODEOWNERS file
```

Comment:
Let's default this based on a relative path from the script instead of depending on System.DefaultWorkingDirectory.

Reply:
You mean "../../../.github/CODEOWNERS"
Why the relative path beat the system default working directory?

Reply:
Because we want this script to be runnable locally as well. I would suggest `$PSScriptRoot/../../../.github/CODEOWNERS`